test/integration/test_util_scripts.py

```diff
@@ -136,45 +141,55 @@ def test_dataset_to_plate(self):
 
         # create several test images in a dataset
         dataset = self.make_dataset("dataset_to_plate-test", client=client)
-        n = 10
+        # Images will be sorted by name and assigned Column first
         image_ids = []
-        for i in range(n):
+        for i in image_names:
             # x,y,z,c,t
-            image = self.create_test_image(100, 100, 1, 1, 1, session)
+            image = self.create_test_image(100, 100, 1, 1, 1, session, name=i)
             self.link(dataset, image, client=client)
             image_ids.append(image.id.val)
 
-        # run the script twice. First with all args...
+        # We run the script twice (parametrize). First with minimum args...
```

Comment:
I was a bit confused by this comment. For each parametrized test, we only run the script right, is that correct? Only the parameters passed to each execution depend on the setup.

Reply:
Simplified comment.